mobile-app/app/screens/WalletNavigator/screens/CreateWallet/CreateWalletGuidelinesV2.tsx

```diff
@@ -0,0 +1,120 @@
+  title: string
+}
+
+const guidelines: GuidelineItem[] = [
+  {
+    title: 'Write the words on paper. Take note of their correct spelling and order.',
+    icon: 'edit-2'
+  },
+  {
+    title: 'Secure them in a safe place. Store them offline at a place only you have access. Keep them private and do not share it with anyone.',
+    icon: 'lock'
+  }
+]
+
+export function CreateWalletGuidelines ({ navigation }: Props): JSX.Element {
+  const [isEnabled, setIsEnabled] = useState(false)
+  const toggleSwitch = (): void => setIsEnabled(previousState => !previousState)
+  const { isLight } = useThemeContext()
+
+  return (
+    <ThemedScrollView
+      dark={tailwind('bg-mono-dark-v2-100')}
+      light={tailwind('bg-mono-light-v2-100')}
+      contentContainerStyle={tailwind('pt-12 px-5 pb-16')}
+      style={tailwind('flex-1')}
+    >
+      <View style={tailwind('flex flex-row justify-center')}>
+        <Image
+          style={{ width: 200, height: 136 }}
+          source={isLight ? LightNewWallet : DarkNewWallet}
+        />
+      </View>
+      <ThemedText
+        style={tailwind('mt-7 text-base text-center')}
+        light={tailwind('text-mono-light-v2-900')}
+        dark={tailwind('text-mono-dark-v2-900')}
+      >
+        {translate('screens/Guidelines', 'You will be shown 24 recovery words on the next screen. Keep your 24-word recovery safe as it will allow you to recover access to the wallet')}
+      </ThemedText>
+      <ThemedText
+        style={tailwind('mt-2 text-base text-center font-semibold')}
+        light={tailwind('text-mono-light-v2-900')}
+        dark={tailwind('text-mono-dark-v2-900')}
+      >
+        {translate('screens/Guidelines', 'Learn more')}
+      </ThemedText>
+      <View style={tailwind('px-3 mt-12')}>
+        {guidelines.map((g, i) => (
+          <View
+            key={i}
+            style={tailwind('flex-row items-start my-3')}
+          >
+            <ThemedView
+              light={tailwind('bg-mono-light-v2-700')}
+              dark={tailwind('bg-mono-dark-v2-700')}
+              style={tailwind('h-6 w-6 flex-row items-center justify-center rounded-full mt-1')}
+            >
+              <ThemedIcon
+                light={tailwind('text-mono-light-v2-100')}
+                dark={tailwind('text-mono-dark-v2-100')}
+                iconType='Feather'
+                name={g.icon}
+                size={14}
+              />
+            </ThemedView>
+            <ThemedText
+              light={tailwind('text-mono-light-v2-700')}
+              dark={tailwind('text-mono-dark-v2-700')}
+              style={tailwind('text-xs ml-4')}
+            >
+              {translate('screens/Guidelines', g.title)}
+            </ThemedText>
+          </View>
+       ))}
+        <View style={tailwind('flex-row items-start mt-3')}>
+          <Checkbox
+            value={isEnabled}
+            onValueChange={toggleSwitch}
+            style={tailwind('h-6 w-6 mt-1 rounded')}
```

Comment:
not sure if possible to change the checkmark color based on theme, currently it is always white, but figma has black in dark mode 
<img width="72" alt="image" src="https://user-images.githubusercontent.com/40191153/175853296-1f1095ce-c0e0-495e-9aa2-10ea3e8f088f.png">

Reply:
Currently There is no option to change checkmark color based on theme

Reply:
can you check with design team and see if they are with this?

Reply:
Confirmed with design team, will be using white tick